app/experimenter/experiments/constants/nimbus.py

```diff
@@ -101,50 +101,25 @@ class NimbusTargetingConfig:
     ),
 )
 
-TARGETING_ALL_ENGLISH = NimbusTargetingConfig(
-    name="All English users",
-    slug="all_english",
-    description="All users in en-* locales.",
-    targeting="localeLanguageCode == 'en'",
```

Comment:
And now if an experiment owner wants to target the old `localeLanguageCode == 'en'`, they'd need to explicitly select all 3 English* locales, correct? "English (British)", "English (Canadian)", and "English (US)"

That seems to give me the following JEXL (when I have "US users (en)" Advanced Targeting on stage):

```jexl
'app.shield.optoutstudies.enabled'|preferenceValue &&
  localeLanguageCode == 'en' &&
  region == 'US' &&
  locale in ['en-CA', 'en-GB', 'en-US']
```

Reply:
Exactly 👍